docs/UC_CV_3ElectrodeCell.md

```diff
@@ -0,0 +1,40 @@
+Use Case: Cyclic Voltammetry on the JUAMI Potentiostat
+Intended User: Undergraduate Researcher at a University in Africa with Introductory Knowledge of Electrochemistry and Cyclic Voltammetry. The instructions in this UC assume that the student already has an electrochemical cell set up and ready for testing (working electrode, counter electrode, and possibly reference electrode are all in a proper beaker).
+As a test case - Mike will perform these instructions as someone with knowledge of electrochemistry but little knowledge of Python.
+
+Experimental Setup:
+
+1. Mike plugs the USB cable into the computer and the potentiostat
+1. Mike attaches the red wire to the working electrode (material of interest)
+1. Mike attaches the black clip to the counter electrode
+1. Mike attaches the white clip to the reference electrode if he is using one, or he attaches it to the counter electrode for a two electrode test
+
+Program Setup:
+1. Mike opens the Pytentiostat script (the script is incomplete and requires filling in of user-defined information)
+1. Mike types a file name for the sample being tested next to 'file name' on line # (file will contain the voltage, current, and time information)
```

Comment:
Ah, sorry, I think I misunderstood.  we are working with a setup script or config file.

To be clear in the UC I would say above something like:
```
1. Mike opens the template experiment config file in a text editor
1. Mike saves the template to a new name in the working directory
1. Mike looks for ``<?>" everywhere in the file and replaces it with the requested information 
1. Mike saves the file
```